docs/articles/documentation/getting-started/README.md

```diff
@@ -82,6 +82,8 @@ TestCafe automatically opens the chosen browser and starts test execution within
 > Important! Make sure to keep the browser tab that is running tests active. Do not minimize the browser window.
 > Inactive tabs and minimized browser windows switch to a lower resource consumption mode
 > where tests are not guaranteed to execute correctly.
+>
+> Do not adjust page zoom during tests. Stable functionality on zoomed in our out pages is not guaranteed.
```

Comment:
I'm not sure you 'adjust zoom' when you zoom in or out. You can 'adjust font' for instance, but zoom... You simply zoom I think.

And 'functionality' is more often used to say 'features' rather than 'operation'. So I read 'stable functionality' as 'features not in beta' at first.

That said, I'd write something like:

```
> Do not zoom pages while testing. Tests may be unstable if the page is zoomed in or out.
```

I got rid of `...in or out pages...` and passive voice in the second sentence. That's also a plus.

I used an `-ing` in the first sentence to avoid repeating 'tests' - not so good concerning guidelines, but the phrase has improved overall IMHO.

Reply:
Applied, thanks